examples/webserver-ts/index.ts

```diff
@@ -29,7 +29,7 @@ nohup python -m SimpleHTTPServer 80 &`;
 const computeInstance = new google.compute.v1.Instance("instance", {
     project: project,
     zone: zone,
-    machineType: `projects/${project}/zones/${zone}/machineTypes/f1-micro`,
+    machineType: "f1-micro",
```

Comment:
Out of scope for this PR, but will we make this into an enum?

Reply:
Any idea where to source it from? I don't think it's defined in the discovery docs.

Reply:
I'm not sure. I poked around on their docs pages for a few mins and it looks like the options are pretty complicated and differ by region. It seems like there has to be a source of truth somewhere, but it's fine to leave for now.